builder/Dockerfile

```
@@ -1,55 +1,59 @@
-# SPDX-License-Identifier: GPL-3.0-or-later
-# author  : paulfantom
-# author  : Austin S. Hemmelgarn <austiN@netdata.cloud>
-
 # This image is used to speed up build process for official netdata images.
+#
+# Copyright: 2019 and later Netdata Incorporated
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+# Author : Paul Emm. Katsoulakis <paul@netdata.rocks>
+# Author : Austin S. Hemmelgarn <austiN@netdata.cloud>
 
-FROM alpine:3.18 as builder
+FROM debian:12 as builder
 
 LABEL org.opencontainers.image.authors="Netdatabot <bot@netdata.cloud>"
 LABEL org.opencontainers.image.source="https://github.com/netdata/helper-images"
 LABEL org.opencontainers.image.title="Netdata Agent Docker Builder Image"
 LABEL org.opencontainers.image.description="Builder image for official Netdata Agent Docker images."
 LABEL org.opencontainers.image.vendor="Netdata Inc."
 
-# Install prerequisites
-RUN apk --no-cache add alpine-sdk \
-                       autoconf \
-                       automake \
-                       bash \
-                       build-base \
-                       bison \
-                       cmake \
-                       curl \
-                       elogind-dev \
-                       flex \
-                       jq \
-                       json-c-dev \
-                       yaml-dev \
-                       libgcrypt-dev \
-                       libmnl-dev \
-                       libtool \
-                       libuuid \
-                       libuv-dev \
-                       lm_sensors \
-                       lz4-dev \
-                       mariadb-dev \
-                       mongo-c-driver-dev \
-                       musl-dev \
-                       musl-fts-dev \
-                       netcat-openbsd \
-                       openssl-dev \
-                       pkgconfig \
-                       postgresql-dev \
-                       protobuf-dev \
-                       py3-pip \
-                       python3 \
-                       python3-dev \
-                       snappy-dev \
-                       util-linux-dev \
-                       zlib-dev \
-                       zstd-dev && \
-    mkdir -p /deps/etc && \
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN chown _apt:root /var/cache/apt /var/lib/apt && \
+    apt-get update && \
+    apt-get upgrade -y && \
+    apt-get install -y --no-install-recommends \
+                    autoconf \
+                    autoconf-archive \
+                    automake \
+                    bison \
+                    build-essential \
+                    cmake \
```

```suggestion
                    build-essential \
                    ca-certificates \
                    cmake \
```

Should fix all `curl` downloads (ebpf, go.d)

---

Need this package in the base too